### PR TITLE
Complete refactorization for css/grayscale.scss and sorted sections in index.html

### PR DESCRIPTION
**Pull request by joariasl**
* Complete refactorization for css/grayscale.scss
* Sections display in sorted of the configuration
* Remove unnecessary id tags
* Update exclusion list
* Update in scripts/newpost
* Added separator in pagination
* Auto toggle to navbar-collapse on ready event. For url of type: file.html#element-id

### CODE REVIEW

**Comment by joariasl** [comment]
Work transmited on [LiveCoding.tv](https://www.livecoding.tv/): https://www.livecoding.tv/video/personal-jekyll-theme-9/
:stuck_out_tongue_winking_eye:

File: css/grayscale.scss [hunk 1]
```diff
@@ -360,10 +282,6 @@ footer {
     background: rgba($light, 0.2);
 }
 
-img {
```
Comment (PanosSakkos) [inline css/grayscale.scss]: We need that, in order to make the images in the posts responsive. It's a fix for issue #36

**Comment by PanosSakkos** [comment]
Apart from the image regression, I the lists in the post don't render correctly.
Actually that was a bug from the Timeline repo that I initially forked and it was tracked by issue #14 :yum: 

Here is the screenshot from my blog when running locally with the new changes

**Comment by joariasl** [comment]
I understand. He had seen that definition.
The correct is to to use `` in the images. But we must maintain backward compatibility.

Do you think putting an extra class to the container of the post and assign the style to their internal images?
```scss
.container-post img {
  display: block;
  max-width: 100%;
  height: auto;
}
```

**Comment by joariasl** [comment]
Added commit in the Pull Request. And deleted unecesary duplication ID in post.html and lastest-post.html.

**Pull request merged**